pkg/istio-agent/xds_proxy.go

```diff
@@ -711,7 +714,7 @@ func (p *XdsProxy) tapRequest(req *discovery.DiscoveryRequest, timeout time.Dura
 	defer p.tapMutex.Unlock()
 
 	// Send to Istiod
-	p.connected.sendRequest(req)
+	connection.sendRequest(req)
```

Comment:
should we be putting it on `requestsChan`? What ha ppens when we hit L717 on a disconnected connection?

it would be good to have a test case for this